#ifndef MAINWINDOW_H
#define MAINWINDOW_H

#include <QMainWindow>
#include "libbremsweg.h"

namespace Ui {
class MainWindow;
}

class MainWindow : public QMainWindow
{
    Q_OBJECT

public:
    explicit MainWindow(QWidget *parent = 0);
    ~MainWindow();

private slots:
    void on_pushButtonBremswegberechnen_clicked();

    void on_checkBoxGefahrenbremsung_stateChanged(int arg1);

    void on_comboBoxFahrzeug_currentIndexChanged(const QString &arg1);

    void on_comboBoxUntergrund_currentIndexChanged(const QString &arg1);

private:
    Ui::MainWindow *ui;
    Bremsweg* bremsweg = new Bremsweg();
    const char* ERROR_FAHRZEUG = "Bitte Fahrzeug ausw채hlen";
    const char* ERROR_UNTERGRUND = "Bitte Untergrund ausw채hlen";
    const char* ERROR_FAHRZEUGundUNTERGRUND = "Bitte Fahrzeug und Untergrund ausw채hlen";
    const char* ERROR_GESCHWINDIGKEIT_NUMMER = "Bitte eine Zahl als Geschwindigkeit eingeben";
    const char* ERROR_GESCHWINDIGKEIT_NEGATIV = "Bitte eine g체ltige Geschwindigkeit eingeben (0..600kmh)";
};

#endif // MAINWINDOW_H
